manifests/server.pp

```diff
@@ -29,7 +29,7 @@
 	$ips = false,	# an optional list of ip's for each in hosts[]
 	$clients = []	# list of allowed client ip's	# TODO: get from exported resources
 ) {
-	$FW = '$FW'			# make using $FW in shorewall easier
+	$fw = '$FW'			# make using $FW in shorewall easier
```

Comment:
The goal of my patch was to fix those errors when you check the code with enabled future parser:

```
Error: Node inheritance is not supported in Puppet >= 4.0.0. See http://links.puppetlabs.com/puppet-node-inheritance-deprecation at ./modules/gluster/vagrant/puppet/manifests/site.pp:10:22
Error: Node inheritance is not supported in Puppet >= 4.0.0. See http://links.puppetlabs.com/puppet-node-inheritance-deprecation at ./modules/gluster/vagrant/puppet/manifests/site.pp:41:28
Error: Node inheritance is not supported in Puppet >= 4.0.0. See http://links.puppetlabs.com/puppet-node-inheritance-deprecation at ./modules/gluster/vagrant/puppet/manifests/site.pp:108:29
Error: Illegal variable name, The given name 'FW' does not conform to the naming rule /^((::)?[a-z]\w*)*((::)?[a-z_]\w*)$/ at ./modules/gluster/vagrant/puppet/manifests/site.pp:137:2
Error: Illegal variable name, The given name 'FW' does not conform to the naming rule /^((::)?[a-z]\w*)*((::)?[a-z_]\w*)$/ at ./modules/gluster/vagrant/puppet/manifests/site.pp:182:22
Error: Illegal variable name, The given name 'FW' does not conform to the naming rule /^((::)?[a-z]\w*)*((::)?[a-z_]\w*)$/ at ./modules/gluster/vagrant/puppet/manifests/site.pp:183:22
Error: Illegal variable name, The given name 'FW' does not conform to the naming rule /^((::)?[a-z]\w*)*((::)?[a-z_]\w*)$/ at ./modules/gluster/vagrant/puppet/manifests/site.pp:187:23
Error: Illegal variable name, The given name 'FW' does not conform to the naming rule /^((::)?[a-z]\w*)*((::)?[a-z_]\w*)$/ at ./modules/gluster/vagrant/puppet/manifests/site.pp:188:22
Error: Illegal variable name, The given name 'FW' does not conform to the naming rule /^((::)?[a-z]\w*)*((::)?[a-z_]\w*)$/ at ./modules/gluster/vagrant/puppet/manifests/site.pp:189:22
Error: Illegal variable name, The given name 'FW' does not conform to the naming rule /^((::)?[a-z]\w*)*((::)?[a-z_]\w*)$/ at ./modules/gluster/vagrant/puppet/manifests/site.pp:193:15
Error: Found 11 errors. Giving up

```